### PR TITLE
feat(transform): dataset.body in starlark is a DataFrame

### PR DESCRIPTION
**Pull request by dustmop**

### CODE REVIEW

**Comment by dustmop** [comment]
Depends on https://github.com/qri-io/starlib/pull/82.

**Review by b5 — COMMENTED** [review]
not finished reviewing, but hey it's a start

File: automation/workflow/wftest/wftest.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func LoadDefaultTestCases() ([]*TestCase, error) {
 								Name:     "transform",
 								Syntax:   "starlark",
 								Category: "transform",
-								Script:   "body = '''a,b,c\n1,2,3\n,4,5,6\n'''\ndef transform(ds,ctx):\n\tds.set_body(body,parse_as='csv')\n",
+								Script:   "body = '''a,b,c\n1,2,3\n4,5,6\n'''\ndef transform(ds,ctx):\n\tds.body.set_csv(body)\n",
```
[the remark this answers is not in the document]
Comment (dustmop) [reply automation/workflow/wftest/wftest.go]: Correct, this is doable, switched all of the tests to stop using `set_csv`.

**Review by b5 — APPROVED** [review]
Looks great. I have a _bunch_ of questions, but I think the smarter thing to do here is merge & file issues on top. Very excited to see DataFrames finally land! Let's get this in!

**Pull request merged**